README.md

```diff
@@ -1,82 +1,209 @@
 
-Either download clone the repository from github and place the variantapi package
-within your code, or do
+There are several ways to create a virtual environment, but you can refer to [pip installation](https://pip.pypa.io/en/stable/installing/) and
+[virtualenv installation](https://virtualenv.pypa.io/en/stable/installation/) to first install these 2 tools if you don't
+have them already installed via a package manager (Linux) or HomeBrew (MacOS), etc.
 
-    pip install https://github.com/saphetor/variant-api-client-python/archive/master.zip
+To create a virtual environment, you can follow the [user guide](https://virtualenv.pypa.io/en/stable/userguide/) or simply run:
 
-### API Documentation
+    virtualenv -p path_to/python3 venv_dir_name
+    
+Activate the virtual environment:
 
-Please visit the [api documentation](http://docs.varsome.apiary.io) to find out how to use the api and
-what values does the api provide as a response to lookup requests
+    source venv_dir_name/bin/activate
 
-### How to get an API key
 
-You generally are not required to have an api key to use the api, though the number of requests you will be able
-to issue will be throttled if they exceed a certain number.
+Finally, to use the client, either download or clone the repository from github and place the `varsome_api` 
+folder inside your project's directory, or run:
 
-Without an API key you will not be able to perform batch requests as well.
+    pip install https://github.com/saphetor/varsome-api-client-python/archive/master.zip
 
-To obtain an API key please [contact us](mailto:support@saphetor.com)
+The client will be installed within your virtual environment. Also, 2 scripts called `varsome_api_run.py` and 
+`varsome_api_annotate_vcf.py` will be available within your virtual environment's `$PATH`.
+
+### Using the scripts to directly annotate a list of variants or a VCF file
+
+#### Annotating a variant or list of variants
+Try the following query to annotate a single variant:
+
+    varsome_api_run.py -g hg19 -q 'chr7-140453136-A-T' -p add-all-data=1
+
+The script should complete without errors and display aproximately 6,700 lines of data from `dann`, `dbnsfp`, `ensembl_transcripts`, `gerp`, `gnomad_exomes`, `gnomad_exomes_coverage`, `icgc_somatic`, `ncbi_clinvar2`, `pub_med_articles`, `refseq_transcripts`, `sanger_cosmic_public`, `uniprot_variants`, `wustl_civic` etc.
+The script can also accept a text file with variants (one per line) and an optional output file to store the
+annotations in. We suggest that you don't use this script for a large number of variants, but use
+the client within your code instead.
+
+    varsome_api_run.py -g hg19 -k api_key -i variants.txt -o annotations.txt -p add-all-data=1
+    
+The command above will read variants from `variants.txt` and dump the annotations to `annotations.txt`. 
+If you don't use the `-k` parameter, the script will do as many requests as there are variants in `variants.txt`,
```

Comment:
For any substantial number of variants you will need to register for an API key. You can try the software without the -k apiKey parameter but you will quickly bump into safeguard limits...

or something like that.